resources/assets/templates/topic/topicCategoryItem.hbs

```diff
@@ -1,18 +1,31 @@
 <div class="col-xs-10 col-sm-10 col-md-10 col-lg-10">
     <div class="category-title">
-    <a href="#categories/{{ model.slug }}/topics"> {{ model.name }} </a>
-    <span class="topic-count"> (Total : {{meta.topicCount}} topics)<span>
-    <br>
-    {{#if meta.topicCount}}
-    <span>Last topics:<span>{{model.value}}<br>
-    {{/if}}
-    <ul>
-        {{#each meta.lastThreeTopics}}
-            <li>
-            <a href="#topics/{{ slug }}"> {{ name }} </a>
-            <span class="category-topic-time">update: {{updated_at}}</span>
-            </li>
-        {{/each}}
-    </ul>
+        <a href="#categories/{{ model.slug }}/topics"> {{ model.name }} </a>
+        <span class="topic-count"> (Total : {{meta.topicCount}} topics)</span>
+        <br>
+        {{#if meta.topicCount}}
+            <span>Last topics:</span>{{model.value}}<br>
+        {{/if}}
+        <ul>
+            {{#each meta.lastThreeTopics}}
+                <li>
+                    <a href="#topics/{{ slug }}"> {{ name }} </a>
+                    <span class="category-topic-time">update: {{updated_at}}</span>
+                </li>
+            {{/each}}
+        </ul>
+
     </div>
-</div>
+    {{#if isAdmin}}
+        <div class="edit-category pull-right">
+            <a href="#topicCategories/{{ model.slug }}/edit">
+                <span class="hidden-xs">Edit category</span>
+                <span class="glyphicon glyphicon-edit" aria-hidden="true"></span>
+            </a>
+            <a class="delete-category-btn">
+                <span class="hidden-xs">Delete category</span>
+                <span class="glyphicon glyphicon-trash" aria-hidden="true"></span>
+            </a>
+        </div>
```

Comment:
Please style it a bit. Maybe move both buttons to the right side and position them one below other